src/.pylintrc-mandatory.jinja

```diff
@@ -51,57 +76,34 @@ enable=anomalous-backslash-in-string,
     method-inverse,
     method-required-super,
     method-search,
-    openerp-exception-warning,
     pointless-statement,
     pointless-string-statement,
     print-used,
     redundant-keyword-arg,
-    redundant-modulename-xml,
     reimported,
-    relative-import,
     return-in-init,
-    rst-syntax-error,
     sql-injection,
     too-few-format-args,
     translation-field,
     translation-required,
     unreachable,
-    use-vim-comment,
-    wrong-tabs-instead-of-spaces,
-    xml-syntax-error
+    use-vim-comment
 {%- if odoo_version >= 15 %},
     attribute-string-redundant,
-    character-not-valid-in-resource-link,
     consider-merging-classes-inherited,
     context-overridden,
-    create-user-wo-reset-password,
-    dangerous-filter-wo-user,
-    dangerous-qweb-replace-wo-priority,
-    deprecated-data-xml-node,
-    deprecated-openerp-xml-node,
-    duplicate-po-message-definition,
```

Comment:
@bizzappdev What's the motivation to introduce new check to version prior to 15.0?

Reply:
Yes indeed. It should have proper condition >=15 and <=18.